TA-defender-atp-status-check/default/props.conf

```diff
@@ -1,8 +1,4 @@
 ###### DefenderATPStatusLog ######
-# Comment below source stanza to not ingest the whole file everytime when the powershell script executes and only ingest when there is update in the status.
-[source::...DefenderATPStatus.log]
-CHECK_METHOD = modtime
-
 
 [DefenderATPStatusLog]
 SHOULD_LINEMERGE = true
```

Comment:
Lets write props with great 8.

SHOULD_LINEMERGE = false (always false)
LINE_BREAKER = regular expression for event breaks - default should work (new line)
DATETIME_CONFIG = NONE
TIME_PREFIX = regex of the text that leads up to the timestamp - No need here
MAX_TIMESTAMP_LOOKAHEAD = how many characters for the timestamp - No need here
TIME_FORMAT = strptime format of the timestamp - No need here
TRUNCATE = 9999
EVENT_BREAKER_ENABLE = true (For UF only - not required here I think)
EVENT_BREAKER = regular expression for event breaks (For UF only - not required here I think)

Reply:
Done